src/Spine/index.js

```diff
@@ -1,10 +1,7 @@
-var PIXI = require('pixi.js'),
-    spine = require('./SpineRuntime');
-
+var PIXI = require('pixi.js');
+var spine = require('../SpineRuntime');
```

Comment:
Later, when going through and writing docs, SpineRuntime could be a "module" name vs "class" name. Also the main index.js for the repo uses it as a module
```
/**
 * @namespace PIXI.spine
 */
module.exports = {
    Spine:          require('./Spine'),
    SpineRuntime:   require('./SpineRuntime')
};
```
For intuitive reasons, I'd say keep it as is, however I agree that at first it looks like a misnomer, however because the word "...Runtime" is used, it denotes that "Spine" is a class specific to pixi and "SpineRuntime" is the module holding the generic classes from Esoteric.